homeassistant/components/skybell/binary_sensor.py

```diff
@@ -16,10 +19,22 @@
 
 SCAN_INTERVAL = timedelta(seconds=10)
 
-# Sensor types: Name, device_class, event
+
+class SensorEntryClass(NamedTuple):
+    """Metadata for sensor types."""
```

Comment:
Just to get it synced up with the other PRs, I would like to suggest something like:
```suggestion
class SkybellBinarySensorMetadata(NamedTuple):
    """Metadata for an individual Skybell binary_sensor."""
```
https://github.com/home-assistant/core/pull/53242/files#diff-2a4673e367cb32db74b949cd9c36f5ad31766f0147dc2673910879daa64b85d4R61-R62

Reply:
done